### PR TITLE
Win32: Implement selection index and child elements for hwnd tabs.

### PR DESCRIPTION
**Pull request by madewokherd**

### CODE REVIEW

**Comment by Conmanx360** [comment]
Looks good to me.

File: xalia/Win32/HwndProvider.cs [hunk 1]
```diff
@@ -365,44 +365,64 @@ private void PollChildren()
 
             var child_hwnds = EnumImmediateChildWindows(Hwnd);
 
-            // First remove any existing children that are missing or out of order
+            // Remove or ignore any existing children
+            HashSet<IntPtr> seen_children = new HashSet<IntPtr>();
             int i = 0;
```
Comment (Conmanx360) [inline xalia/Win32/HwndProvider.cs:394]: Is this a copy paste error? I don't see `seen_children` used here.

**Comment by Conmanx360** [comment]
Other than the `seen_children` comment above, this looks fine to me.

File: xalia/Win32/HwndProvider.cs (hunk 1 above)
Comment (madewokherd) [reply xalia/Win32/HwndProvider.cs:394]: Odd, VS usually warns me about unused things. But yes, that shouldn't be there as I'm pretty sure win32 can't give me duplicate child hwnds.

**Pull request merged**